addons/mail/static/src/components/message/message_tests.js

```diff
@@ -1123,7 +1123,7 @@ QUnit.test('rendering of tracked field with change of value from true to false',
     await this.createMessageComponent(message);
     assert.strictEqual(
         document.querySelector('.o_Message_trackingValue').textContent,
-        "Is Ready:truefalse",
+        "Is Ready:true",
```

Comment:
For boolean tracking `false` is an actual value that has to be shown too.

Note that this was made a translatable string in master, where the problem is not there anymore. Maybe we'd want to backport this.

Reply:
I think what we need to do is to reinroduce the original condition you had on the template (just "if value") and make sure that the JS always returns a string to be displayed when that makes sense, so "0" for numbers and "False" (translatable) for booleans.

Reply:
Not so simple because of javascript ... 
```
!"1" --> true
!"0" -> false
```

So if the number is 0, it still won't show it ..